src/dist/replication/lib/replica_stub.cpp

```diff
@@ -375,6 +385,9 @@ void replica_stub::initialize(const replication_options &opts, bool clear /* = f
     _max_concurrent_bulk_load_downloading_count =
         _options.max_concurrent_bulk_load_downloading_count;
 
+    _abnormal_read_trace_latency_threshold = FLAGS_abnormal_read_trace_latency_threshold;
+    _abnormal_write_trace_latency_threshold = FLAGS_abnormal_write_trace_latency_threshold;
```

Comment:
why are you add `_abnormal_read_trace_latency_threshold`? you can use FLAGS_abnormal_read_trace_latency_threshold directly